colorbleed/scripts/launch.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pprint
+
+import avalon.lib as lib
+
+
+# TODO: Remove redundant hack
+import sys
+sys.path.append(r"P:\pipeline\dev\git\env_prototype")
+import env_prototype.api as api
+
+
+config_root = os.path.dirname(os.path.dirname(__file__))
+os.environ["TOOL_ENV"] = os.path.join(config_root, "environments")
+
+
+def launch(tools, executable, args):
+
+    tools_env = api.get_tools(tools.split(";"))
+    env = api.compute(tools_env)
+
+    env = api.merge(env, current_env=dict(os.environ))
+    print("Environment:\n%s" % pprint.pformat(env, indent=4))
+
+    # Search for the executable within the tool's environment
+    # by temporarily taking on its `PATH` settings
+    original = os.environ["PATH"]
+    os.environ["PATH"] = env.get("PATH", os.environ.get("PATH", ""))
```

Comment:
Let's also get this workaround out, just add in the extra argument for lib.which :)

Reply:
Can we replace this module with the one from our `env_prototype` ( we need to pick that awesome name ;) )